### PR TITLE
Release notes for Oxford and Paris

### PR DESCRIPTION
**Pull request by timothymcmackin**
Update info for Oxford and Paris, based on info in these blog posts:

- https://spotlight.tezos.com/paris-upgrade-live/
- https://spotlight.tezos.com/oxford-2-upgrade-live/

### CODE REVIEW

File: docs/architecture/governance/amendment-history.md [hunk 1]
```diff
@@ -209,3 +209,30 @@ Nairobi's main changes are:
...
 
 For more information, see the blog post from [Nomadic Labs](https://research-development.nomadic-labs.com/nairobi-announcement.html) and the [reference documentation](https://tezos.gitlab.io/protocols/017_nairobi.html).
+
+## [Oxford](https://tezos.gitlab.io/protocols/018_oxford.html) (Proxford)
+
+*Oxford* was autonomously [activated](https://tzkt.io/5070849) on 9 February 2024.
+
+Oxford's main changes are:
+
+* [Refinement of Tezos PoS](https://research-development.nomadic-labs.com/oxford2-announcement.html#automated-staking): the Oxford 2 proposal introduces changes to [slashing](https://research-development.nomadic-labs.com/oxford2-announcement.html#refined-slashing) and an [automated staking mechanism for bakers](https://research-development.nomadic-labs.com/oxford2-announcement.html#automated-staking).
```
Comment (NicNomadic) [inline docs/architecture/governance/amendment-history.md:219]: here and elsewhere, Oxford is no longer a proposal (and drop the "2"):
```suggestion
* [Refinement of Tezos PoS](https://research-development.nomadic-labs.com/oxford2-announcement.html#automated-staking): Oxford introduces changes to [slashing](https://research-development.nomadic-labs.com/oxford2-announcement.html#refined-slashing) and an [automated staking mechanism for bakers](https://research-development.nomadic-labs.com/oxford2-announcement.html#automated-staking).
```

File: docs/architecture/governance/amendment-history.md [hunk 2]
```diff
@@ -209,3 +209,30 @@ Nairobi's main changes are:
...
+Additionally, Oxford 2 [simplifies the deployment of rollups](https://research-development.nomadic-labs.com/oxford2-announcement.html#introducing-private-rollups-and-other-improvements-to-smart-rollups) both on protocol and periodic test networks, as well as on ad-hoc dedicated ones.
+* [Timelocks are re-enabled](https://research-development.nomadic-labs.com/timelocks-are-re-enabled): a new design and implementation of Timelocks addresses security concerns that led to their temporary deactivation in a previous protocol upgrade.
+
+For more information, see the blog post from [Nomadic Labs](https://research-development.nomadic-labs.com/oxford-announcement.html) and the [reference documentation](https://tezos.gitlab.io/protocols/018_oxford.html).
+
+## [Paris](https://tezos.gitlab.io/protocols/019_paris.html) (PtParisB)
+
+*Paris* was autonomously [activated](https://tzkt.io/5726209) on 5 June 2024.
+
+Paris's main changes are:
```
Comment (NicNomadic) [inline docs/architecture/governance/amendment-history.md:231]: Shouldn't this be spelled so:
```suggestion
Paris' main changes are:
```

File: docs/architecture/governance/amendment-history.md [hunk 3]
```diff
@@ -209,3 +209,30 @@ Nairobi's main changes are:
...
+
+* [10-second block time](https://research-development.nomadic-labs.com/10-second-blocktime.html): Lower latency and faster finality on layer 1 without compromising decentralization or security.
```
Comment (NicNomadic) [inline docs/architecture/governance/amendment-history.md:233]: 10-seconds ?

File: docs/architecture/governance/amendment-history.md [hunk 4]
```diff
@@ -209,3 +209,30 @@ Nairobi's main changes are:
...
+* [Data availability layer](https://research-development.nomadic-labs.com/paris-announcement.html#the-dal-activates-on-mainnet-boosting-smart-rollups-capacity): Boosting throughput and scalability of Smart Rollups.
```
Comment (NicNomadic) [inline docs/architecture/governance/amendment-history.md:234]: Too vague. For example:
```suggestion
* [Data availability layer](https://research-development.nomadic-labs.com/paris-announcement.html#the-dal-activates-on-mainnet-boosting-smart-rollups-capacity): Boosting throughput and scalability of Smart Rollups.
It enables Tezos Layer 1 to attest the publication of data living outside Layer 1 blocks, increasing by orders of magnitude the bandwidth of data attested by the Layer 1.
```

File: docs/architecture/governance/amendment-history.md [hunk 5]
```diff
@@ -209,3 +209,30 @@ Nairobi's main changes are:
...
+* [Adaptive issuance, staking, and adaptive slashing](https://research-development.nomadic-labs.com/adaptive-issuance-paris.html): A major overhaul of Tezos Proof-of-Stake
```
Comment (NicNomadic) [inline docs/architecture/governance/amendment-history.md:235]: This gives no clue about what it is!
```suggestion
* [Adaptive issuance, staking, and adaptive slashing](https://research-development.nomadic-labs.com/adaptive-issuance-paris.html): A major overhaul of Tezos Proof-of-Stake, adapting the economics of tez to fit better with real-world usage, and to increase the chain security. The proposed mechanism ties the protocol’s regular issuance of tez to the ratio of staked tez over the total supply, in order to nudge the staked fund ratio towards a protocol-defined target.
```

**Review by NicNomadic — CHANGES_REQUESTED** [review]

File: docs/architecture/governance/amendment-history.md (hunk 2 above)
Comment (timothymcmackin) [reply docs/architecture/governance/amendment-history.md:231]: No, Paris is a singular noun, so it gets "'s" even though it ends in a letter "s." 
https://www.oxfordlearnersdictionaries.com/us/grammar/online-grammar/possessive-s-and-s

File: docs/architecture/governance/amendment-history.md (hunk 5 above)
Comment (timothymcmackin) [reply docs/architecture/governance/amendment-history.md:236]: I shouldn't have copied directly from the blog post! :-)

File: docs/architecture/governance/amendment-history.md (hunk 3 above)
Comment (timothymcmackin) [reply docs/architecture/governance/amendment-history.md:233]: Not sure. I'd say "The Tezos block time is 10 seconds," but here "10-seconds" is being used as a modifier. Can't think of any related examples...

File: docs/architecture/governance/amendment-history.md (hunk 2 above)
Comment (NicNomadic) [reply docs/architecture/governance/amendment-history.md:231]: OK, I didn't know that, thanks. (I see that other guidelines allow both James' and James's)

File: docs/architecture/governance/amendment-history.md (hunk 5 above)
Comment (NicNomadic) [reply docs/architecture/governance/amendment-history.md]: You could have, but based on the other blog <https://research-development.nomadic-labs.com/paris-announcement.html>

File: docs/architecture/governance/amendment-history.md (hunk 3 above)
Comment (NicNomadic) [reply docs/architecture/governance/amendment-history.md:233]: ok, I trust you :-)

File: docs/architecture/governance/amendment-history.md [hunk 6]
```diff
@@ -209,3 +209,31 @@ Nairobi's main changes are:
...
+
+## [Oxford](https://tezos.gitlab.io/protocols/018_oxford.html) (Proxford)
+
+*Oxford* was autonomously [activated](https://tzkt.io/5070849) on 9 February 2024.
+
+Oxford's main changes are:
+
+* [Refinement of Tezos PoS](https://research-development.nomadic-labs.com/oxford2-announcement.html#automated-staking): Oxford introduces changes to [slashing](https://research-development.nomadic-labs.com/oxford2-announcement.html#refined-slashing) and an [automated staking mechanism for bakers](https://research-development.nomadic-labs.com/oxford2-announcement.html#automated-staking).
+The latter aims to smoothen the transition towards a new staked funds management API and avoids manual bookkeeping to counter over-delegation.
+* [Private rollups](https://research-development.nomadic-labs.com/oxford2-announcement.html): Oxford 2 introduces private Smart Rollups, allowing developers to choose between permissionless or permissioned deployments.
```
Comment (NicNomadic) [inline docs/architecture/governance/amendment-history.md:221]: forgot to remove the "2" here:
```suggestion
* [Private rollups](https://research-development.nomadic-labs.com/oxford2-announcement.html): Oxford introduces private Smart Rollups, allowing developers to choose between permissionless or permissioned deployments.
```

File: docs/architecture/governance/amendment-history.md [hunk 7]
```diff
@@ -209,3 +209,31 @@ Nairobi's main changes are:
...
+Additionally, Oxford 2 [simplifies the deployment of rollups](https://research-development.nomadic-labs.com/oxford2-announcement.html#introducing-private-rollups-and-other-improvements-to-smart-rollups) both on protocol and periodic test networks, as well as on ad-hoc dedicated ones.
```
Comment (NicNomadic) [inline docs/architecture/governance/amendment-history.md:222]:
```suggestion
Additionally, Oxford [simplifies the deployment of rollups](https://research-development.nomadic-labs.com/oxford2-announcement.html#introducing-private-rollups-and-other-improvements-to-smart-rollups) both on protocol and periodic test networks, as well as on ad-hoc dedicated ones.
```

**Review by NicNomadic — APPROVED** [review]
LGTM

**Pull request merged**